curriculum/challenges/english/01-responsive-web-design/basic-css/change-the-color-of-text.english.md

```diff
@@ -29,7 +29,7 @@ tests:
   - text: Your <code>h2</code> element should have a <code>style</code> declaration.
     testString: assert($("h2").attr('style'));
   - text: Your <code>h2</code> element should be red.
```

Comment:
```suggestion
  - text: Your <code>h2</code> element should have the color <code>red</code>.
```
If we're only going to accept `color: red` I think there should be a stronger hint that that's the case.

Reply:
How about this @ojeytonwilliams ?
```suggestion
  - text: Your <code>h2</code> element should have the color set to <code>red</code>.
```

Reply:
Yeah, that works for me.